src/main/java/uk/gov/ch/model/capital/StatementOfCapital.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.ch.model.capital;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class StatementOfCapital {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("class_of_shares")
+    private String classOfShares;
+
+    @JsonProperty("currency")
+    private String currency;
+
+    @JsonProperty("number_allotted")
+    private int numberAllotted;
+
+    @JsonProperty("aggregate_nominal_value")
+    private int aggregateNominalValue;
+
+    @JsonProperty("prescribed_particulars")
+    private int prescribedParticulars;
```

Comment:
Should this be an int? Looking at the story it seems to correspond to a string of text